SK.Application/Profiles/Queries/DetailsProfile/DetailsProfileQueryHandler.cs

```diff
@@ -12,22 +13,29 @@ namespace SK.Application.Profiles.Queries.DetailsProfile
     public class DetailsProfileQueryHandler : IRequestHandler<DetailsProfileQuery, ProfileDto>
     {
         private readonly IApplicationDbContext _context;
+        private readonly IMapper _mapper;
 
-        public DetailsProfileQueryHandler(IApplicationDbContext context)
+        public DetailsProfileQueryHandler(IApplicationDbContext context, IMapper mapper)
```

Comment:
You are injecting AutoMapper, but you are not using it anywhere.

Reply:
Done.